raiden/app.py

```diff
@@ -127,7 +140,29 @@ def app(privatekey, eth_rpc_endpoint, registry_contract_address,
     config = App.default_config.copy()
     config['host'] = listen_host
     config['port'] = listen_port
-    config['privatekey_hex'] = privatekey
+
+    accmgr = AccountManager(keystore_path)
+    if not accmgr.address_in_keystore(address):
+        addresses = list(accmgr.accounts.keys())
```

Comment:
Sorry, if I have been unclear.The addresses are displayed to the user later on. I think it would make sense to sort the addresses list here in order to have deterministic output between different runs of the program.